fastai/learner.py

```diff
@@ -14,6 +14,18 @@
 
 class Learner():
     def __init__(self, data, models, opt_fn=None, tmp_name='tmp', models_name='models', metrics=None, clip=None):
+        """
+        Learner is a class that combines a ModelData object with a nn.Module object, such that you can train that
+        module.
+        data (ModelData): An instance of ModelData.
+        models: chosen neural architecture for solving a supported problem.
+                Eg. ResNet-34
```

Comment:
'ResNet-34' would not be a valid input here. If giving an example of param use, be sure it's a valid input for that param.

Reply:
Interesting. I was trying to mimic a similar param I saw in the library. I think this makes sense, I might have misread the example took from. Thanks for clarification.

Reply:
As a point of clarification, models represents a nn.Module correct?